calour/database.py

```diff
@@ -99,6 +98,7 @@ def add_terms_to_features(exp: Experiment, dbname, use_term_list=None, field_nam
         None to get default type
     ignore_exp : list of int or None, optional
         list of experiments to ignore when adding the terms
+    **kwargs: database specific additional parameters (see database interface get_feature_terms() function for specific terms)
```

Comment:
could you provide a reference to link the function (using the rst syntax as did in doc in other functions)?

Reply:
I think it's more problematic since it depends on the database modules you have installed. So the kwargs are different between dbbact_calour and gnps_calour. Also, these modules reside in different github repos, so don't know how to do it...